xml/systemd.xml

```diff
@@ -14,61 +14,58 @@
   </dm:docmanager>
  </info>
  <para>
-  The program &systemd; is the process with process ID 1. It is responsible for
-  initializing the system in the required way. &systemd; is started directly by
+  The program &systemd; is responsible for initializing the system in the
+  required way and has the process ID 1. &systemd; is started directly by
   the kernel and resists signal&nbsp;9, which normally terminates processes.
-  All other programs are either started directly by systemd or by one of its
-  child processes.
+  All other programs are either started directly by &systemd; or by one of its
+  child processes. &systemd; is a replacement for the System V init daemon and
+  fully compatible with System V init (by supporting init scripts).
  </para>
  <para>
-  Systemd is a replacement for the System V init daemon. &systemd; is fully
-  compatible with System V init (by supporting init scripts). One of the main
-  advantages of systemd is that it considerably speeds up boot time by
-  aggressively paralleling service starts.  Furthermore, systemd only starts a
+  The main advantage of &systemd; is that it considerably speeds up boot time by
+  aggressively paralleling service starts. Furthermore, &systemd; only starts a
   service when it is really needed. Daemons are not started unconditionally at
-  boot time, but rather when being required for the first time. systemd also
-  supports Kernel Control Groups (cgroups), snapshotting and restoring the
-  system state and more. See <link
-  xlink:href="http://www.freedesktop.org/wiki/Software/systemd/"/> for details.
+  boot time, but when being required for the first time. &systemd; also
+  supports Kernel Control Groups (cgroups), snap-shooting and restoring the
```

Comment:
snapshotting is correct. Although _creating snapshots_ is better.

Reply:
oh yes, it take creating snapshots then